testutil/integration/options.go

```diff
@@ -1,14 +1,30 @@
 package integration
 
+var (
+	RunUntilNextBlockOpts = RunOptions{
+		WithAutomaticCommit(),
+		WithAutomaticFinalizeBlock(),
+	}
+)
+
 // RunConfig is the configuration for the testing integration app.
 type RunConfig struct {
 	AutomaticFinalizeBlock bool
 	AutomaticCommit        bool
+	ErrorAssertion         func(error)
 }
 
 // RunOption is a function that can be used to configure the integration app.
 type RunOption func(*RunConfig)
 
+// RunOptions is a list of RunOption. It implements the Append method for convenience.
+type RunOptions []RunOption
+
+// Append returns a new RunOptions with the given RunOptions appended.
+func (runOpts RunOptions) Append(newRunOpts ...RunOption) RunOptions {
```

Comment:
Will there be a use case for `RunOptions` and it `Append` method?

Reply:
No longer. I've removed `RunOption` and `RunConfig` altogether.  :sweat_smile: